odpf/optimus/core/v1beta1/runtime.proto

```diff
@@ -670,6 +678,23 @@ message JobStatusResponse {
     repeated JobStatus statuses = 1;
 }
 
+message JobRunRequest {
+    string project_name = 1;
+    string job_name = 2;
+    google.protobuf.Timestamp start_date = 3;
+    google.protobuf.Timestamp end_date = 4;
+    repeated string filter = 5;
+}
+
+message JobRunResponse {
+    repeated RunStatus run_status = 1;
+}
+
+message RunStatus {
```

Comment:
Can we use JobRun instead

Reply:
new rpc is named as `JobRun`, is it okay to have same name for rpc and message?

Reply:
I believe that should be fine, we don't have a technical limitation right

Reply:
No, I think it will make more sense to end user. as API response will look like below :
```
{
    "jobRun": [
        {
            "state": "success",
            "scheduledAt": "2022-03-01T02:00:00Z"
        },
        {
            "state": "success",
            "scheduledAt": "2022-03-02T02:00:00Z"
        }
    ]
}
```

Reply:
RunStatus is more generic, we have replay runs, backup runs, job runs. Lets keep it specific.
Curious to understand how the jobRun name is coming if we are using RunStatus as the message.

Reply:
Agree, that's why I have changed the message to` Jobrun` at the latest commit. Just gave an example earlier regarding the response structure with `jobRun`